ibm/mas_devops/roles/db2/defaults/main.yml

```diff
@@ -12,6 +11,9 @@ db2_channel: "{{ lookup('env', 'DB2_CHANNEL') }}"
 
 db2_instance_name: "{{ lookup('env', 'DB2_INSTANCE_NAME') }}" # e.g. db2u-iot or db2u-manage
 db2_dbname: "{{ lookup('env', 'DB2_DBNAME') | default('BLUDB', true) }}"
+db2_type: "{{ lookup('env', 'DB2_TYPE') | default('db2wh', true) }}" # e.g. db2wh or db2oltp
+db2_timezone: "{{ lookup('env', 'DB2_TIMEZONE') }}"
```

Comment:
Are we missing a readme entry for db2_timezone?

Reply:
thanks! just fixed !